dwave/embedding/chimera.py

```diff
@@ -74,7 +76,30 @@ def find_clique_embedding(k, m, n=None, t=None, target_edges=None):
 
     m, n, t, target_edges = _chimera_input(m, n, t, target_edges)
 
-    embedding = processor(target_edges, M=m, N=n, L=t).tightestNativeClique(len(nodes))
+    # Special cases to return optimal embeddings for small k.  The general clique embedder uses chains of length
+    # at least 2, whereas cliques of size 1 and 2 can be embedded with single-qubit chains.
+
+    if len(nodes) == 1:
+        # If k == 1 we simply return a single chain consisting of a randomly sampled qubit.
+
+        qubits = set()
+        for edge in target_edges:
+            qubits = qubits.union(edge)
+
+        qubit = np.random.choice(list(qubits))
```

Comment:
It's actually faster to use the built in `random`:
```
qubit = random.choice(tuple(qubits))
```

Reply:
Good suggestion, thanks!